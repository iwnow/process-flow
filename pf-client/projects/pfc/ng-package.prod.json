{
  "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
  "dest": "../../dist/pfc",
  "lib": {
    "entryFile": "src/public_api.ts"
  }
}
