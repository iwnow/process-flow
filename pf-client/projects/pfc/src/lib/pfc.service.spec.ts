import { TestBed, inject } from '@angular/core/testing';

import { PfComponentsService } from './pfc.service';

describe('PfComponentsService', () => {
	beforeEach(() => {
		TestBed.configureTestingModule({
			providers: [PfComponentsService]
		});
	});

	it('should be created', inject([PfComponentsService], (service: PfComponentsService) => {
		expect(service).toBeTruthy();
	}));

	it('snap test', () => {
		const o = {
			k: 5
		};

		o.k++;

		expect(o).toMatchSnapshot();

	});
});
