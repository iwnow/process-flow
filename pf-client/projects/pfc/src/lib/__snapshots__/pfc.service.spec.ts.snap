// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PfComponentsService snap test 1`] = `
Object {
  "k": 6,
}
`;
